### PR TITLE
Created database changelog file.

### PR DESCRIPTION
**Pull request by KR4K0**

### CODE REVIEW

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 1]
```diff
@@ -23,16 +23,42 @@ databaseChangeLog:
...
-  - changeSet:
-      id: 2
-      author: daniel-kujawa
+              - column:
+                  name: role
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+ 
       changes:
```
Comment (kujawad) [inline src/main/resources/db/changelog/db.changelog-master.yaml]: According to YAML specification, this is `key`, so, in each set (in this case `- changeSet`), the key (`changes`) cannot be duplicated. Remove this line.

**Review by kujawad — CHANGES_REQUESTED** [review]

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 2]
```diff
@@ -23,16 +23,42 @@ databaseChangeLog:
...
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: description
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                   name: issuer_id
```
Comment (krzjez) [inline src/main/resources/db/changelog/db.changelog-master.yaml]: For me, issuer_id and tester_id make no sense. Tester IS the issuer in our project. I suggest to change the names of these 2 columns to author_id and developer_id.

File: src/main/resources/db/changelog/db.changelog-master.yaml (hunk 1 above)
Comment (KR4K0) [reply src/main/resources/db/changelog/db.changelog-master.yaml]: Done

File: src/main/resources/db/changelog/db.changelog-master.yaml (hunk 2 above)
Comment (KR4K0) [reply src/main/resources/db/changelog/db.changelog-master.yaml]: Done as suggested.

**Review by kujawad — APPROVED** [review]

**Pull request merged**